aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -187,6 +187,18 @@ public class AwsProperties implements Serializable {
    */
   public static final String GLUE_CATALOG_ENDPOINT = "glue.endpoint";
 
+  /**
+   * If set, Glue will always update the catalog table if the table already exists in glue catalog.
+   * By default, Glue catalog will only be able to create new table and will throw
+   * AlreadyExistsException when register an existing table name.
+   */
+  public static final String GLUE_CATALOG_FORCE_REGISTER_TABLE = "glue.force-register-table";
+
+  public static final boolean GLUE_CATALOG_FORCE_REGISTER_TABLE_DEFAULT = false;
+
+  /** Configure the Glue Catalog S3 FileIO Region to allow cross region s3 access */
+  public static final String GLUE_CATALOG_FILE_IO_REGION = "glue.catalog-file-io-region";
```

Comment:
So we already have `client.region` which sets the region for all services. I am going to be introducing a change in the near future that will allow setting per-region for the default AWS Client Factory (and we can extend it to the Assume Role client factory as well), so we probably want a more general per-service naming scheme. Based on how existing parameters are formatted where `glue.` and `s3.` are already established prefixes, most likely something like: `glue.region`, `s3.region`, `kms.region`, etc...